mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -263,6 +264,36 @@ static CrashReportLogLevel CheckCrashReportLog (string directory_str, bool clear
 			}
 		}
 
+		static string get_breadcrumb_value (string file_prefix, string directory_str, bool clear)
+		{
+			var allfiles = Directory.GetFiles(directory_str, $"{file_prefix}_*" );
```

Comment:
How would this work if you had crashes from 2 sessions?

Reply:
You can run 2 instances of VSMac at once, so you could _technically_ have that happen.

Reply:
Would they be set up to use the a common directory for breadcrumbs? If so, we should probably start using pid-coded directories for these.

Reply:
How do you solve this issue for e.g. `Ide.log`?

Reply:
We use a DateTime formatted in the log file, based on when the log file is created.

Reply:
@Therzok which is the session ID? The IDs I see in EnableMicrosoftTelemetry seem more static (like BundleID). It needs to be something that's available from the beginning as these breadcrumbs are create at any and all phase. 

If we do have that, we can add that to the breadcrumbs' names (e.g. `crash_stage_3_53221`). But let's do that separately from this PR, since this is a problem that already existed (stage breadcrumbs have the same problem)